### PR TITLE
Workflow to auto-generate vanity imports HTML meta-tags

### LINKED ISSUE #746: Automate vanity import path generation
### Problem description: 

When a new go submodule is added to the project the go-import meta tags should be updated to reflect that in order to serve requests for the module.
Currently, the HTML meta tags are generated in https://github.com/fybrik/vanity-imports repository by triggering a workflow which uses a tool called vangen (https://github.com/leighmcculloch/vangen) where the packages paths are hard-coded in vangen.json .

We need to automate the generation of the modules list in vangen.json and trigger that upon a push to the go repos that use vanity import.

### Possible solutions:

Add a new workflow in every go repos in the organization that uses vanity import which will trigger a workflow in vanity-imports repo upon a push.
The workflow in the vanity-imports repo will run a script to auto-generate the modules list and run vangen tool to create the meta-tags for them based on the json file.  Lastly, pushing the meta-tags to site branch as is done now.

Alternatively, a new github workflow can be added to every go repos in the organization that uses vanity import which will use vangen tools in https://github.com/fybrik/vanity-imports repository.

### PR DESCRIPTION
**Pull request by revit13**
This PR adds a workflow to auto-generate  HTML meta-tags using the tools in https://github.com/fybrik/vanity-imports repository.
The workflow is triggered on every push to the master branch.
If the HTML meta-tags were changed it creates a PR against site branch.

Closes #746

### CODE REVIEW

File: .github/workflows/vanity-imports.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Fybrik Golang Vanity Imports
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
```
Comment (roee88) [inline .github/workflows/vanity-imports.yaml:8]: Is it needed?

File: .github/workflows/vanity-imports.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        run: |
+          cp vanity-imports/hack/tools/vangen.sh fybrik-master-branch
+          cd fybrik-master-branch; ./vangen.sh; cd ..
+          cp fybrik-master-branch/vangen.json vanity-imports/
+          make -C vanity-imports generate
+      - name: Copy vanity import meta-tags to fybrik-site-branch
+        run: |
+          rm -rf fybrik-site-branch/fybrik
+          cp -r vanity-imports/vangen/fybrik fybrik-site-branch
+      - name: Create Pull Request
```
Comment (roee88) [inline .github/workflows/vanity-imports.yaml:38]: Once we're sure it works then maybe commit and push directly?

**Review by roee88 — APPROVED** [review]
Great, please do the same for the other repos.

File: .github/workflows/vanity-imports.yaml (hunk 2 above)
Comment (revit13) [reply .github/workflows/vanity-imports.yaml:38]: sure we can...

File: .github/workflows/vanity-imports.yaml (hunk 1 above)
Comment (revit13) [reply .github/workflows/vanity-imports.yaml:8]: I am not sure... I saw it in other workflows...

Comment (roee88) [reply .github/workflows/vanity-imports.yaml:8]: Imports won't change between master and tags

File: .github/workflows/vanity-imports.yaml (hunk 2 above)
Comment (roee88) [reply .github/workflows/vanity-imports.yaml:38]: So let's try with PR first and then see how it goes

**Comment by revit13** [comment]
Taking the approach of adding the meta-tags to 404.html file

**Pull request closed without merging**